pennylane_lightning/src/simulators/lightning_kokkos/algorithms/AdjointJacobianKokkos.hpp

```diff
@@ -202,12 +154,13 @@ class AdjointJacobian final
                                              ops.getOpsWires()[op_idx],
                                              !ops.getOpsInverses()[op_idx]) *
                         (ops.getOpsInverses()[op_idx] ? -1 : 1);
-
                     for (size_t obs_idx = 0; obs_idx < num_observables;
                          obs_idx++) {
+                        const size_t idx =
+                            trainableParamNumber + obs_idx * tp_size;
+                        // trainableParamNumber * num_observables + obs_idx;
```

Comment:
Is this needed?

Reply:
No, good spot. This was the old indexing pattern.